docs/avoid-additional-properties.md

```diff
@@ -0,0 +1,45 @@
+# AvoidAdditionalProperties
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Policy-V1-03
+
+## Output Message
+
+The properties of type "Addiotinal Properties" are not allowed
+
+## Description
+
+If a property is of type additionalProperties then this should be flagged as an error
```

Comment:
Change this to something like, "Definitions must not have properties named additionalProperties."